libs/src/services/creatorNode/index.js

```diff
@@ -45,15 +45,20 @@ class CreatorNode {
    * Pulls off the user's clock value from a creator node endpoint and the user's wallet address.
    * @param {string} endpoint content node endpoint
    * @param {string} wallet user wallet address
+   * @param {number} timeout max time alloted for clock request
+   * @param {Object?} [params={}] optional query string params
    */
-  static async getClockValue (endpoint, wallet, timeout) {
+  static async getClockValue (endpoint, wallet, timeout, params = {}) {
+    let baseReq = {
+      url: `/users/clock_status/${wallet}`,
+      method: 'get',
+      baseURL: endpoint,
+      timeout
```

Comment:
should only add this field if provided, not sure what happens if it is passed in as `undefined`, [axios defaults to 0](https://github.com/axios/axios#request-config)